common/config/azure-pipelines/ci.yaml

```diff
@@ -2,12 +2,13 @@ jobs:
 - job: PRBuild
   condition: succeeded()
   strategy:
-    maxParallel: 2

```

Comment:
If `maxParallel` is not specified, the default behavior is to run all matrix rows in parallel.